docs/wiki/extra-documents/regarding-foundations.md

```diff
@@ -13,7 +13,7 @@ September 13, 2018: There are no current or future plans to create an official G
 ## Selected experiences of other projects
 
 * **Bitcoin** Foundation has been [mired in controversies](https://en.wikipedia.org/wiki/Bitcoin_Foundation) regarding its leadership and received cease and desist letters from US institutions.
-* The **Tezos** Foundation ended up in a [vicious legal dispute](https://www.wired.com/story/tezos-blockchain-love-story-horror-story/) with the project founders whilst some $400m in funds at the time were locked.
+* The **Tezos** Foundation ended up in a vicious legal dispute with the project founders whilst some $400m in funds at the time were locked.
```

Comment:
Why remove the link?

Reply:
At least from my end, this link is behind a paywall and therefore not accessible for me.

Reply:
Indeed.